manifests/04_deploy.yaml

```diff
@@ -18,8 +18,8 @@ spec:
     spec:
       serviceAccountName: openshift-service-cert-signer-operator
       containers:
-      - name: operator
-        image: openshift/origin-service-serving-cert-signer:latest
+      - name: openshift-service-cert-signer-operator
+        image: quay.io/openshift/origin-service-serving-cert-signer:latest
```

Comment:
This is the string that should show up in `image-references` under the tag.  You don't need to change the container name.